Update accelerators.rst (#45549)
ray-project/ray: doc/source/ray-core/scheduling/accelerators.rst

            For example, ``NEURON_RT_VISIBLE_CORES=1,3 ray start --head --resources='{"neuron_cores": 2}'``
            lets Ray only see devices 1 and 3.

>             Amazon also maintains a very good documentation and examples gallery for getting started on Ray on Neuron with the help of EKS as a orchestration substrate. You can find that here: `https://awslabs.github.io/data-on-eks/docs/category/inference-on-eks` <https://awslabs.github.io/data-on-eks/docs/category/inference-on-eks>

Reviewer: LGTM; leave it to @angelinalg for the content

nit: break long line into small chunk

>             Amazon also maintains a very good documentation and examples gallery for getting started on Ray on Neuron with the help of EKS as a orchestration substrate. You can find that here: `https://awslabs.github.io/data-on-eks/docs/category/inference-on-eks` <https://awslabs.github.io/data-on-eks/docs/category/inference-on-eks>

Reviewer 2: ```suggestion
            See the `Amazon documentation<https://awslabs.github.io/data-on-eks/docs/category/inference-on-eks>` for more examples of Ray on Neuron with EKS as an orchestration substrate.
```
